projects/kit/components/arrow/arrow-options.ts

```diff
@@ -22,3 +29,10 @@ export const TUI_ARROW_OPTIONS = new InjectionToken<TuiArrowOptions>(
         factory: () => TUI_ARROW_DEFAULT_OPTIONS,
     },
 );
+
+export const TUI_ARROW_MODE: InjectionToken<TuiArrowMode> = new InjectionToken(
+    'Type of icon in dropdowns for interactive or disable mode',
+    {
+        factory: () => ({interactive: TUI_ARROW, disabled: ''}),
```

Comment:
Let's make it visible in disabled state as well and override it in our internals. I believe most people would like to have the old behavior, before we implemented this change.

Reply:
fixed